src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php

```diff
@@ -38,6 +38,15 @@ class Translator extends BaseTranslator implements WarmableInterface
     private $resourceLocales;
 
     /**
+     * Holds parameters from addResource() calls so we can add these resources
+     * after the ones passed in the resource_files option (which are added
+     * lazily in loadResources() as well).
+     *
+     * @var array
+     */
+    private $otherResources = array();
```

Comment:
Yes, using this array to collect `addResource()` calls and actually execute them after the `resource_files` is ugly, but we don't have access to the private `\Symfony\Component\Translation\Translator::$resources` field so that we could "unshift" the `resource_files` in front of it in `loadResources`.